relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -465,6 +467,12 @@ pub fn extract_tags(
                 let tag_value = if *cache_hit { "true" } else { "false" };
                 span_tags.insert(SpanTagKey::CacheHit, tag_value.to_owned());
             }
+            if let Some(cache_keys) = span.data.value().and_then(|data| data.cache_key.value()) {
+                span_tags.insert(
+                    SpanTagKey::CacheKey,
+                    serde_json::to_string(cache_keys).unwrap(),
```

Comment:
Let's not unwrap here, even if the risk of panicking is low (e.g. `if let Ok(cache_keys) = cache_keys`).